scipy/integrate/_quadpack_py.py

```diff
@@ -641,6 +641,38 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
     romb : integrator for sampled data
     scipy.special : for coefficients and roots of orthogonal polynomials
 
+
+    Notes
+    -----
+
+    **Details of QUADPACK level routines**
+
+    `quad` calls routines from the FORTRAN library QUADPACK. This section
+    provides details on the conditions for each routine to be called and a
+    short description of each routine. For each level of integration qagse
+    is used for finite limits or qagie is used, if either limit (or both!) are
+    infinite. The following provides a short desciption from [1]_ for each
+    routine.
```

Comment:
```suggestion
    short description of each routine. For each level of integration, qagse
    is used for finite limits, or qagie is used if either limit (or both!) are
    infinite. The following provides a short description from [1]_ of each
    routine.
```
Also consider double backticks around `qaqse` and `qagie` to distinguish from English.
(similarly below)